test/integration/storage.integration.test.js

```diff
@@ -81,9 +116,41 @@ describe.only("uploadDocumentAtId", () => {
     const uploaded = uploadDocumentAtId(document, queueNumber);
     await expect(uploaded).rejects.toThrow("Document is not valid");
   });
+
+  it("should allow user to specify ttl", async () => {
+    isValid.mockReturnValueOnce(true);
+    const { id: queueNumber } = await getQueueNumber();
+    const document = { foo: "bar" };
+    const uploaded = await uploadDocumentAtId(document, queueNumber, 20000);
+    expect(uploaded).toMatchObject(thatIsUploadResponse);
```

Comment:
create matchers for upload response with TTL and retrieved document with TTL so that the test failures are explicit about ttl being missing